installer/templates/new/widgets/text/text.js

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import Widget from '../../assets/javascripts/widget';
+import {updatedAt} from '../../assets/javascripts/helpers';
 
 import './text.scss';
 
 Widget.mount(class Text extends Widget {
   render() {
+    let status = "";
+    if (this.state.widgetStatus) {
+      status = this.state.widgetStatus;
+    }
     return (
-      <div className={this.props.className}>
-        <h1 className="title">{this.props.title}</h1>
+      <div className={`${this.props.className} ${status}`}>    
```

Comment:
Currently there are only 2 classes which could work with this defined in application.scss `status-warning` and `status-danger`.

The job file shouldn't be too coupled with the exact class names of the ui.
I'd prefer to have the status class calculated in a fuction instead from within `render`.


```javascript
#  either as an one-liner
status() { return ((s) s ? `status-${s}` : "")(this.state.status) }

# Or in more readable form
status() {
  if (!this.state.status) { return ""; }
  
  return`status-${this.state.status}`;
}
```